src/mainboard/sifive/hifive/src/main.rs

```diff
@@ -176,20 +176,28 @@ fn test_ddr(addr: *mut u32, size: usize, w: &mut print::WriteTo) -> Result<(), (
 
 // TODO: move out of mainboard
```

Comment:
Would this be a good time to move this to the device_tree crate?

Reply:
Done (I decided to do this as separate commit in this PR).